data/hai/hai reveal 6 end.txt

```diff
@@ -317,25 +335,45 @@ mission "Hai Reveal [C05] Cleanup"
 			`	She gives you a hard stare, but her resolve isn't up to the task for more than a couple seconds. She looks away, and you can see her blinking away more tears. "Is this to be my end, then?" she asks. "Is this the price I pay for my efforts?"`
 			label decide
 			choice
-				`	(Spare her.)`
-					goto spare
-				`	(End her.)`
-					goto end
 				`	"Am I to be judge, jury, and executioner?"`
 					goto judge
+					to display
+						not "Hai Reveal [C05] Cleanup: label judge"
+				`	(Spare her.) "The whole galaxy hunts you now. Let that be your justice."`
+					goto spare
+				`	(End her.) "The price you pay is justice."`
+					goto end
+				`	(Spare her.) "I've sinned more than you. If people like us decided what is right or wrong, the galaxy would be in cinders."`
+					goto spare
+				`	(End her.) "This is for all the trouble you put me through."`
+					goto end
+				`	(Spare her.) "Consider this my movie review."`
+					goto spare
+					to display
+						has "Hai Reveal [C05] Cleanup: label judge"
+				`	(End her.) "Consider this my movie review."`
+					goto end
+					to display
+						has "Hai Reveal [C05] Cleanup: label judge"
 			label spare
-			`	You draw your weapon and point it at her, your thumb on the safety. When you look inside yourself though, you think maybe the future is worth the sacrifice made in the attempt.`
+			`	You draw your weapon and point it at her, your thumb on the safety. When you look inside yourself, though, you think: "Maybe the future is worth the sacrifice made in the attempt."`
```

Comment:
Can I summon @EjoThims to this specific line? I'm thinking maybe to avoid the clunky punctuation we should go more like:
```suggestion
			`	You draw your weapon and point it at her, your thumb on the safety. When you look inside yourself though, you wonder if perhaps the future is worth the sacrifice made in the attempt.`
```

Reply:
Yup. This was the best I could do with this line, but what does it mean?

Reply:
@UnorderedSigh The most recent suggestion is much better punctuation, but it is still a confusing wording. Especially in reference to which sacrifice and attempt.

I get what it’s referring to, just not what it’s trying to say about what it’s referring to.

Reply:
The sentence confuses me too, so I've been wary of touching it.

Reply:
> The most recent suggestion is much better punctuation

I already applied the most recent suggestion.